docs/tutorials/dicom.ipynb

```diff
@@ -105,7 +105,7 @@
       "source": [
         "## Overview\n",
         "\n",
-        "This tutorial shows how to use [dicom](https://github.com/tensorflow/io/tree/master/tensorflow_io/dicom) in TensorFlow IO to decode DICOM files with TensorFlow."
+        "This tutorial shows how to use [decode_dicom_image](https://www.tensorflow.org/io/api_docs/python/tfio/image/decode_dicom_image) in TensorFlow IO to decode DICOM files with TensorFlow."
```

Comment:
This should autolink when you wrap the API symbol in backticks: \`tfio.image.decode_dicom_image\`